charts/ceph-csi-rbd/Chart.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 appVersion: canary
 description: "Container Storage Interface (CSI) driver,
-provisioner, snapshotter, and attacher for Ceph RBD"
+provisioner, snapshotter, resizer and attacher for Ceph RBD"
```

Comment:
same change is required in cephfs chart also.

Reply:
@Madhu-1 I have done that on attacher removal PR # https://github.com/ceph/ceph-csi/pull/3149/commits/d86508274c75174787abf7d83e2c5056b77a5db1

Reply:
@humblec can you add that to this PR as it makes sense to make the same changes in the similar PR?

Reply:
@Madhu-1 Done.

Reply:
@humblec i dont see the changes yet, please check again.

Reply:
@humblec any specific reason to remove cephfs change from this PR?

Reply:
@Madhu-1 nothing has to be changed there https://github.com/ceph/ceph-csi/blob/devel/charts/ceph-csi-cephfs/Chart.yaml ?

Reply:
I missed the mentioned PR already merged.